script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt

```diff
@@ -672,7 +686,7 @@ and the Principle of 'Four Limbs'..."
 [sha:356e6a35d464ffb753c366c20dac91c40dedaa33]{
 -- Page 617, Offset 18134.
 -- 教会はひとりの祖を封印するため、100年単位で準備しては実行していたけど、成功例はたったの二回。
-The time the Church dedicates to preparations for sealing a solitary Ancestor is denominated in centuries,
+The Church takes centures in preparing and carrying out the sealing of even just a single Ancestor,
```

Comment:
Kinda like the orig here

Reply:
I don't at all.

Reply:
Alright, but it reads a little stiff to me - maybe tweak
> The Church takes centuries to prepare and carry out the sealing of even just a single Ancestor,